src/cli-storage.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "fs";
+import os from "os";
+
+const __confDir = os.homedir() + "/.baseplate";
+if (!fs.existsSync(__confDir)) {
+  fs.mkdirSync(__confDir);
+}
+
+export async function storageProvider() {
+  // @ts-ignore
+  const conf = await import("conf");
```

Comment:
I haven't used `conf` before - looks good for this use case. Why isn't it imported as a static import at the top of the file?

Reply:
I faced the  'moduleResolution' error:
```
There are types at 'node_modules/conf/dist/source/index.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.
```

and found these:
https://github.com/sindresorhus/conf/issues/175
https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

changing 'moduleResolution' breaks other stuff